pkg/types/codec.go

```diff
@@ -11,6 +11,7 @@ const (
 	ErrNotASlice       = InvalidArgumentError("element is not a slice")
 )
 
+//go:generate mockery --quiet --name Encoder --output ./mocks/ --case=underscore
```

Comment:
We really should consider using a mockery config file. I think @jmank88 played with this a bit already?

Reply:
https://github.com/smartcontractkit/chainlink-common/pull/607

Reply:
Oh wait, are these new mocks being used in this repo? If not, they should be created from the repos that use them instead.

Reply:
@jmank88  wdym by 
> they should be created from the repos that use them instead.
How does that work?